src/test/javascript/spec/component/tutorial-groups/course-tutorial-groups/course-tutorial-groups-overview.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CourseTutorialGroupsOverviewComponent } from 'app/overview/course-tutorial-groups/course-tutorial-groups-overview/course-tutorial-groups-overview.component';
+import { Component, Input } from '@angular/core';
```

Comment:
Unused import.

Reply:
Removed in 404a61f9e625e0e40fb810485d800baaea28b685